config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+    invited:
+      createaccounttitle: "Creare un profilo"
+      createaccountbody: "Per creare un nuovo profilo, inserisci il tuo indirizzo e-mail e una password. Il vostro indirizzo e-mail non sarà mai mostrato agli utenti e sarà utilizzato solo se avrai bisogno di azzerare la tua password, o per ricevere delle notifiche opzionali. Il tuo nome utente potrà essere cambiato successivamente."
+      invitedby: "Invitato da :"
+      username: "Nome utente:"
+      email: "Indirizzo email :"
+      password: "Password :"
+      passwordagain: "Password (di conferma):"
+      about: "Vuoi dirci qualcosa di te? Questo è opzionale, ma sarà visibile sul tuo profilo pubblicamente e potrà essere cambiato in qualunque momento."
+      abouttitle: "A riguardo:"
+      markdownavailable: "Formattazione Markdown disponibile"
+      signup: "Iscriversi"
+    invite:
+      passinvitation: "Condividi un invito a tua volta"
+      passinvitationbody: "Questa comunità potrà crescere solo se i suoi membri inviteranno nuovi utenti, come è stato fatto con te. Pensaci un attimo e invita qualcuno di tua conoscenza"
+      passinvitationor: "o"
+      passinvitationskip: "In seguito"
+  stories:
+    edit:
+      edit: "Modifica l'articolo"
+      merge: "Accorpa a:"
+      shortid: "L'id breve dell'articolo cui si vuole accorpare"
+      unavailable: "Non disponibile :"
+      unavailableat: "URL sorgente non disponibile, abilita la visualizzazione del testo nella cache"
+      modreason: "Motivazione della moderazione: "
+      markdown: "Disponibile formattazione markdown"
+      save: "Salva"
+      cancel: "Annullare le modifiche"
+    form:
+      alreadysubmitted: "Errore: questo articolo è già stato pubblicato "
+      alreadysubmittedflash: "Nota: Questo articolo è gia stato pubblicato %{timeago}, ma può essere ripubblicato"
+      previousdiscussion: "Per favore consulta <a href=\"%{storypath}\" target=\"_blank\"> la discussione precedente</a> per questo articolo."
+      previousexpireddiscussion: "Per favore consulta prima <a href=\"%{storypath}\" \"target=_blank\">la discussione precedente</a> per questo articolo.  Se il contenuto è cambiato o giustifica una nuova discussione, puoi ripubblicarlo."
+      fetchtitle: "Recuperare il titolo"
+      url: "URL :"
+      title: "Titolo :"
+      changetitle: "Gli utenti hanno suggerito di modificare il titolo di questo articolo in:"
+      tags: "Etichette :"
+      changetags: "Gli utenti di hanno suggerito di modificare le etichette di questo articolo in:"
+      text: "Testo :"
+      placeholdertext: "Opzionale quando si pubblica un URL; per favore consulta le linee guida"
+      storysubmissionguidelines: "Linee guida per la pubblicazione degli articoli"
+      guidelinespart1: "Non modificare il titolo, se non quando il titolo originale dell'articolo manca di contestualizzazione o è confusionario. <strong>Per favore rimuovi dal titolo componenti estranei come il nome del sito o della sezione.</strong>"
```

Comment:
estranee

Reply:
Stando alla voce [Treccani](http://www.treccani.it/vocabolario/componente/) (indice 3), è corretto il maschile.